public/js/selfoss-events-entries.js

```diff
@@ -151,8 +151,9 @@ selfoss.events.entries = function(e) {
             dataType: 'json',
```

Comment:
After loads the last page of the feed, this request is fired on every scroll event. Probably because now the `stream-more` button is always present: https://github.com/niol/selfoss/blob/2620ee0c2a17ae44e4fffb63968f1cd5a23b3312/public/js/selfoss-events-entries.js#L130